src/anomalib/data/avenue.py

```diff
@@ -197,13 +197,13 @@ class Avenue(AnomalibVideoDataModule):
 
     def __init__(
         self,
-        root: Path | str,
-        gt_dir: Path | str,
+        root: Path | str = "./datasets",
+        gt_dir: Path | str = "",
```

Comment:
I think the ground truth gets downloaded to `"./datasets/avenue/ground_truth_demo"` by default. Maybe we could specify that path here?